tests/ReportbackApiTest.php

```diff
@@ -43,11 +44,11 @@ public function testCreatingNewReportback()
             'quantity'         => $this->faker->numberBetween(10, 1000),
             'why_participated' => $this->faker->paragraph(3),
             'num_participants' => null,
-            'file_id' => $this->faker->randomNumber(4),
-            'caption' => $this->faker->sentence(),
-            'source' => 'runscope',
-            'remote_addr' => '207.110.19.130',
-            'file' => $file,
+            'file_id'          => $this->faker->randomNumber(4),
```

Comment:
Is StyleCI ok with this spacing? I think I get errors when I try to align arrow notation.